pandas/core/ops/__init__.py

```diff
@@ -600,6 +600,7 @@ def _combine_series_frame(left, right, func, axis: int, str_rep: str):
 
     Returns
     -------
+    new_data : DataFrame or Dict[int, Series]
```

Comment:
woa really? this is strange to return different types; also is result now superfluousz?

Reply:
> woa really? this is strange to return different types

we _could_ continue calling left._construct_result here (so we would conitnue returning only DataFrame) and then below just re-call _constuct_result.  But there isnt a nice option to never return DataFrame

> also is result now superfluousz?

good catch, will update